src/FeatureIntroLearnMoreLink/index.tsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx jsx */
+import * as React from "react";
+import * as typography from "../typography";
+import { jsx } from "@emotion/core";
+import { colors } from "../colors";
+import { useFeatureIntroControlInternalContext } from "../shared/FeatureIntroControlContext";
+
+type Props = Pick<
+  React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+  "className" | "style" | "id"
+>;
+
+/**
+ * Component to render Feature Intro learn more button.
+ *
+ * If this component is rendered in the children of `FeatureIntroControl`, then
+ * `FeatureIntroControl` will render this element in it's layout. Otherwise, it's
+ * rendered as-is.
+ */
+export const FeatureIntroLearnMoreLink: React.FC<
+  React.HTMLProps<HTMLAnchorElement> | Props
+> = (id) => {
+  const { setLearnMoreLink } = useFeatureIntroControlInternalContext();
+
+  const element = React.useMemo(
+    () => (
+      <div id={id && `${id}-learn-more-link`}>
```

Comment:
ahh I think these will need to still use the root id for the generated value
```suggestion
  const { setLearnMoreLink, id: contextId } = useFeatureIntroControlInternalContext();

  const element = React.useMemo(
    () => (
      <div id={id ?? contextId && `${contextId}-learn-more-link`}>
```

Reply:
🤦